develop.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Develop | OrnitheMC</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="develop.js" defer></script>
+</head>
+
+<body>
+    <h1>Develop</h1>
+
+    <h2>Getting started</h2>
+    <p>
+        If you are updating the Loom or Ploceus version in <code>build.gradle</code> file, make sure their version match
+        (example, Ploceus of version <code>1.5-SNAPSHOT</code> needs Loom to be of version <code>1.5-SNAPSHOT</code> as
+        well)</p>
+    <p>TODO</p>
+
+    <h2>Latest versions</h2>
+
+    <input type="checkbox" id="allow-snapshots" />
+    <label for="allow-snapshots">Include Snapshots</label>
+
+    <fieldset>
+        <legend>Select a ModLoader</legend>
+        <input type="radio" id="mod-loader-fabric" name="modLoader" checked />
+        <label for="mod-loader-fabric">Fabric</label>
+
+        <input type="radio" id="mod-loader-quilt" name="modLoader" />
+        <label for="mod-loader-quilt">Quilt</label>
+    </fieldset>
+
+    <br />
+
+    <input type="text" id="mc-version" list="version-list" value="1.7.2" />
+    <label for="mc-version-autocomplete">Game version</label>
+    <datalist id="version-list">
+        <option value="1.7.2" />
+        <!-- DO NOT INSERT ANYTHING HERE -->
+        <!-- Data here will be inserted by JavaScript -->
+    </datalist>
+    <br />
+
+    Paste these in gradle.properties in your project
+
+    <br />
+
+    <code id="ornithe-dependencies">
```

Comment:
> which stuff ?
This can probably be changed to a `<pre>` block, and a style can be applied to it on the style.css file to differentiate it from the rest of the page, like a box with a dark gray background with a margin

Reply:
this should be good now, will probably need to change some more style later on, but for now this works